guides/micronaut-health-endpoint/micronaut-health-endpoint.adoc

```diff
@@ -10,6 +10,8 @@ common:create-app-features.adoc[]
 
 === Testing Health Endpoints
 
+The Micronaut management dependency added for the project supports monitoring your application via endpoints: special URIs that return details about the health and state of your application. Once the `management` dependency is included a `/health` endpoint is exposed.
```

Comment:
can you show the dependency added. Example in liquibase section: 

https://guides.micronaut.io/latest/micronaut-liquibase-gradle-java.html#database-migration-with-liquibase